app/models.py

```diff
@@ -38,3 +39,19 @@ class HungerGameFilter(str, Enum):
     @staticmethod
     def list():
         return [c.value for c in HungerGameFilter]
+
+
+def facet_plural(facet: str):
+    p = inflect.engine()
+    plural = p.plural(facet)
+    if facet == "packaging":
+        facet_plural = facet
+    else:
+        facet_plural = plural
```

Comment:
Why do we handle differently for packaging ?

```suggestion
    facet_plural = plural
    # special case for packaging ?
    if facet == "packaging":
        facet_plural = facet
```

Reply:
because that how the search api takes it